tests/test_pr_comments.py

```diff
@@ -215,6 +221,73 @@ def test_has_contractor_comment_no_comments(app, reqctx, requests_mocker):
     assert result is False
 
 
+def test_internal_cherry_pick_comment(app):
+    pr = make_pull_request(user='nedbat')
+    open_edx_release = "Matt's Open edX Release"
+    question = "This is a question"
+    answer = "This is an answer"
+    with app.test_request_context('/'):
+        comment = github_internal_cherrypick_comment(pr, open_edx_release, question, answer)
+    assert "Matt's Open edX Release" in comment
+    assert "This is a question" in comment
+    assert "This is an answer" in comment
+    assert "perhaps this PR should be cherry picked" in comment
+
+def test_open_edx_dry_run_environment_var_set(app, reqctx, requests_mocker):
+    test_open_edx_release = 'mduboseedx,nedbat,fakeuser'
+    with mock.patch.dict('os.environ', {'TEST_OPENEDX_RELEASE_NAME': test_open_edx_release}):
+        pr = make_pull_request(user='nedbat')
+        requests_mocker.post(
+            "https://api.github.com/repos/edx/edx-platform/issues/1/comments",
+            headers={"Content-Type": "application/json"},
+        )
+        with reqctx:
+           app.preprocess_request()
+           result = openedx_webhooks.tasks.github.pull_request_opened(pr)
+        assert result[1] is True
+        assert "Test Release" in requests_mocker.last_request.json()['body']
+
+def test_open_edx_dry_run_environment_var_set_user_not_listed(app, reqctx, requests_mocker):
+    test_open_edx_release = 'mduboseedx,fakeuser'
+    with mock.patch.dict('os.environ', {'TEST_OPENEDX_RELEASE_NAME': test_open_edx_release}):
+        pr = make_pull_request(user='nedbat')
+        requests_mocker.post(
+            "https://api.github.com/repos/edx/edx-platform/issues/1/comments",
+            headers={"Content-Type": "application/json"},
+        )
+        with reqctx:
+            app.preprocess_request()
+            result = openedx_webhooks.tasks.github.pull_request_opened(pr)
+        assert result[1] is False
+        assert requests_mocker.last_request.text is None
```

Comment:
I did not use `requests_mocker.called` since that should evaluate to True as there are other calls to the people.yaml and org.yaml files